src/modelmanager.cpp

```diff
@@ -434,6 +429,26 @@ static Status processCustomNodeConfig(const rapidjson::Value& nodeConfig, Custom
 }
 
 #if (MEDIAPIPE_DISABLE == 0)
+bool ModelManager::CheckStartFromGraph(std::string inputPath, MediapipeGraphConfig& mpConfig, bool checkModelMeshPath) {
```

Comment:
This should be refactored out, later - now logic of parsing MP configs is partially here, paritally in mdeiapipegraphconfig so this is spaghetti now. Conditional approval